.github/workflows/main-api.yml

```
@@ -0,0 +1,23 @@
+
+    deploy:
+        name: ????
+        needs: test
+        uses: ./.github/workflows/deploy.yml
+        secrets:
+          HOST: ${{ secrets.MAIN_API_ADDRESS }}
+          USERNAME: ${{ secrets.MAIN_API_USERNAME }}
+          PASSWORD: ${{ secrets.MAIN_API_PASSWORD }}
+          SSH_PORT: ${{ secrets.MAIN_API_SSH_PORT }}
+        with:
+          environment: main-api
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)